sklearn/ensemble/tests/test_weight_boosting.py

```diff
@@ -33,10 +33,10 @@
 
 # Toy sample
 X = [[-2, -1], [-1, -1], [-1, -2], [1, 1], [1, 2], [2, 1]]
-y_class = ["foo", "foo", "foo", 1, 1, 1]    # test string class labels
+y_class = ["foo", "foo", "foo", "bar", "bar", "bar"]
```

Comment:
To resolve `test_classification_toy`. Numpy now warns when running `np.array(["foo", 1])`.